Kconfig

```diff
@@ -2,6 +2,24 @@ mainmenu "SOF $(PROJECTVERSION) Configuration"
 
 comment "Compiler: $(CC_VERSION_TEXT)"
 
+menu "CAVS"
+
+config HP_MEMORY_BANKS
+	int "HP memory banks count"
+	depends on CAVS
+	default 8
+	help
+	  Available memory banks count for High Performace memory
```

Comment:
One minor change, I would add here more detail on the memory banks. i.e. the bank size and that disabling unused banks would lower power consumption. Ditto for cores.

Reply:
also worth mentioning that reducing the number of banks too low could result in not enough free space for base FW image.

Reply:
Added